### PR TITLE
Very important bugfix

### LINKED ISSUE #816: tabbed notes/instructions breaks
**Describe the bug**

if i click the tabs they break and become like this
image

**To Reproduce**

1. enable `Project notes tabs`
2. visit any project
3. click a tab
4. see broken layout

**Expected behavior**

tabs work properly

**Screenshots**

before clicking:
image
after clicking tab:
image

**Device (please complete the following information):**

- OS: windows 10 2004
- Browser chrome
- Version latest master branch as of posting this 2322c665ce26954c2fc628619c2280680df1544a

**Additional context**

enabled addons (i think)
```yaml
[
  {
    "id": "scratch-messaging",
    "name": "Scratch Messaging"
  },
  {
    "id": "msg-count-badge",
    "name": "Message count on extension icon"
  },
  {
    "id": "60fps",
    "name": "60FPS player mode"
  },
  {
    "id": "block-switching",
    "name": "Block switching"
  },
  {
    "id": "clones",
    "name": "Clone counter"
  },
  {
    "id": "cloud-games",
    "name": "Cloud games"
  },
  {
    "id": "editor-colored-context-menus",
    "name": "Colored context menus"
  },
  {
    "id": "editor-devtools",
    "name": "Developer tools"
  },
  {
    "id": "discuss-button",
    "name": "Discuss button"
  },
  {
    "id": "search-load-more-fix",
    "name": "Fix \"Load more\" scrolling in search results"
  },
  {
    "id": "forum-search",
    "name": "Forum Search"
  },
  {
    "id": "image-uploader",
    "name": "Forums image uploader"
  },
  {
    "id": "last-edit-tooltip",
    "name": "Last edit tooltip"
  },
  {
    "id": "more-links",
    "name": "More links"
  },
  {
    "id": "mouse-pos",
    "name": "Mouse position"
  },
  {
    "id": "pause",
    "name": "Pause button"
  },
  {
    "id": "project-notes-tabs",
    "name": "Project notes tabs"
  },
  {
    "id": "progress-bar",
    "name": "Project progress bar"
  },
  {
    "id": "remix-tree-button",
    "name": "Remix tree button on project pages"
  },
  {
    "id": "scratchr2",
    "name": "Scratch 2.0 → 3.0"
  },
  {
    "id": "search-profile",
    "name": "Search profile"
  },
  {
    "id": "editor-searchable-dropdowns",
    "name": "Searchable dropdowns"
  },
  {
    "id": "full-signature",
    "name": "Show full areas"
  },
  {
    "id": "studio-tools",
    "name": "Studio manager tools"
  }
]
```

### PR DESCRIPTION
**Pull request by TheColaber**
Resolves #816 

**Changes**

I forgot @WorldLanguages told me to use .style... so it messed up my ugly code and now I fixed it again.

Also, apple removed an important piece in the project-info addon so I added those back too.

### CODE REVIEW

File: addons/project-notes-tabs/style.css [hunk 1]
```diff
@@ -29,3 +29,7 @@
 .sa-selected {
   background-color: rgba(77, 151, 255, 0.1);
 }
+
+.project-description {
+  height: -webkit-fill-available;
```
Comment (WorldLanguages) [inline addons/project-notes-tabs/style.css:34]: We support Firefox as well

File: addons/project-info/userscript.js [hunk 2]
```diff
@@ -23,8 +23,16 @@ export default async function ({ addon, console, msg }) {
         projectInfo = { spriteCount: "?", scriptCount: "?" };
       }
     }
-    container.appendChild(msg(projectInfo.spriteCount == 1 ? "sprite" : "sprites", { num: projectInfo.spriteCount }));
+    container.appendChild(
+      document.createTextNode(
+        msg(projectInfo.spriteCount == 1 ? "sprite" : "sprites", { num: projectInfo.spriteCount })
```
Comment (WorldLanguages) [inline addons/project-info/userscript.js:28]: How this is done will change after the next l10n PR is merged, so I'm keeping this PR pending for now.

File: addons/project-notes-tabs/style.css (hunk 1 above)
Comment (TheColaber) [reply addons/project-notes-tabs/style.css:34]: What does ff use?

Comment (WorldLanguages) [reply addons/project-notes-tabs/style.css:34]: I'd bet `-moz`, but you'd need to do your research (and test it!)

Comment (TheColaber) [reply addons/project-notes-tabs/style.css:34]: I don't have firefox sadly but I'll fix this.

File: addons/project-info/userscript.js (hunk 2 above)
Comment (TheColaber) [reply addons/project-info/userscript.js:28]: What needs be changed here?

**Review by WorldLanguages — APPROVED** [review]

**Pull request merged**